enabler/src/de/schildbach/pte/TexasProvider.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @author Ojas Ahuja
+ */
+package de.schildbach.pte;
+
+import de.schildbach.pte.dto.Product;
+import de.schildbach.pte.dto.Style;
+import de.schildbach.pte.dto.Style.Shape;
+
+public class TexasProvider extends AbstractNavitiaProvider
+{
+	private static String API_REGION = "us-tx";
+
+	public TexasProvider(final String apiBase, final String authorization)
+	{
+		super(NetworkId.TX, apiBase, authorization);
+
+		setTimeZone("America/Chicago");
```

Comment:
@schildbach this was intentional. The string passed to setTimeZone appears to be based off of [the tz format](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which only includes Chicago for Central Time in the US (UTC-5).